File: aoj/volume100/scala/A10008.scala
object Main {
  def main(args: Array[String]) {
    val sc = new java.util.Scanner(System.in)
    val a = sc.nextInt
    val b = sc.nextInt
    printf("%d %d %.6f", (a / b), (a % b), (a.toDouble / b.toDouble))
  }
}

File: aoj/volume100/java/A10008.java
import java.util.Scanner;

class A10008 {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        int a = scan.nextInt();
        int b = scan.nextInt();

        System.out.printf("%d %d %.6f\n",
                a / b,
                a % b,
                (double)a / (double)b);
    }
}
